src/Firehose.Web/Authors/RicardoPrestes.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Firehose.Web.Infrastructure;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+
+namespace Firehose.Web.Authors
+{
+	public class RicardoPrestes: IAmACommunityMember, IFilterMyBlogPosts
+	{
+		public string FirstName => "Ricardo";
+		public string LastName => "Prestes";
+		public string StateOrRegion => "Cornélio Procópio, Brasil";
+		public string EmailAddress => "ricardo.logan@hotmail.com";
+		public string ShortBioOrTagLine => "Mobile developer, Burger Monkeys Co-founder";
+		public Uri WebSite => new Uri("https://oficinadologan.wordpress.com/");
+		public string TwitterHandle => "ricardo_prestes";
+		public string GitHubHandle => "ricardoprestes";
+		public string GravatarHash => "9802e38d5bd2cd85db8b0720d5feed29";
+		public IEnumerable<Uri> FeedUris
+		{
+			get { yield return new Uri("https://oficinadologan.wordpress.com/rss"); }
+		}
+		public GeoPosition Position => new GeoPosition(-23.194571, -50.7795215);
+		public string FeedLanguageCode => "pt";
+
+		public bool Filter(SyndicationItem item) =>
```

Comment:
We already provide a better default filter for Xamarin

Reply:
Removed